acceptance-tests/tests/workflows/secretsFlow.spec.ts

```diff
@@ -11,6 +11,20 @@ const SECRET = {
   value: 'an initial secret value',
 };
 
+const secretPollingOptions = {
+  interval: 2000,
+  timeout: 20000,
```

Comment:
Still polling 10 times, just spacing them out over a wider wide to give the secrets a better chance of existing